setup.py

```diff
@@ -55,7 +60,7 @@ def read(*parts):
                         **extra_extension_args)
 
 if cythonize:
-    [udunits_ext] = cythonize(udunits_ext)
+    [udunits_ext] = cythonize(udunits_ext, compiler_directives={'linetrace': True})
```

Comment:
We could also add the directive: `binding=True`. That can sometime help with cython code coverage.